### PR TITLE
update prod ci/cd

### PR DESCRIPTION
**Pull request by thepsalmist**
## Description

Charterafrica CI/CD for prod deployment, testing the deployment on an amd64 instance

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: .github/workflows/charterafrica-deploy-prod.yml [hunk 1]
```diff
@@ -11,9 +11,9 @@ concurrency:
 
 env:
   DOKKU_REMOTE_BRANCH: "master"
```
Comment (kilemensi) [inline .github/workflows/charterafrica-deploy-prod.yml:13]: Lets delete `DOKKU_REMOTE_BRANCH`... we're no longer pushing to a branch.

File: .github/workflows/charterafrica-deploy-prod.yml [hunk 2]
```diff
@@ -11,9 +11,9 @@ concurrency:
...
-  DOKKU_REMOTE_URL: "ssh://dokku@charter.prod.codeforafrica.org/charterafrica"
+  DOKKU_REMOTE_URL: "ssh://dokku@ui-1.prod.codeforafrica.org/codeforafrica-ui"
```
Comment (kilemensi) [inline .github/workflows/charterafrica-deploy-prod.yml:14]: Why `codeforafrica-ui`? The app should be named `charteafrica-ui-staging`, `charterafrica-ui-something`.

`codeforafrica-ui` reminds me of https://cfa.dev.codeforafrica.org/

File: .github/workflows/charterafrica-deploy-prod.yml [hunk 3]
```diff
@@ -86,7 +86,7 @@ jobs:
           cache-to: type=local,dest=/tmp/.buildx-cache-new
           context: .
           file: ./Dockerfile.charterafrica
-          platforms: linux/arm64
+          platforms: linux/amd64
```
Comment (kilemensi) [inline .github/workflows/charterafrica-deploy-prod.yml:89]:
```suggestion
```

Isn't `linux/amd64` the default (at least charterafrica-deploy-dev.yml thinks o).

File: apps/charterafrica/contrib/dokku/Dockerfile [hunk 4]
```diff
@@ -1 +1 @@
-FROM codeforafrica/charterafrica-ui:0.0.15
+FROM codeforafrica/charterafrica-ui:0.0.16
```
Comment (kilemensi) [inline apps/charterafrica/contrib/dokku/Dockerfile:1]:
```suggestion
FROM codeforafrica/charterafrica-ui:0.0.21
```

I think we've already create images up to `0.0.19`/`0.0.20`. Lets not go back.

**Review by kilemensi — COMMENTED** [review]
👍🏽

File: apps/charterafrica/contrib/dokku/Dockerfile (hunk 4 above)
Comment (thepsalmist) [reply apps/charterafrica/contrib/dokku/Dockerfile:1]: Moved to0.0.22, we have 0.0.21

File: .github/workflows/charterafrica-deploy-prod.yml [hunk 5]
```diff
@@ -10,10 +10,9 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  DOKKU_REMOTE_BRANCH: "master"
-  DOKKU_REMOTE_URL: "ssh://dokku@charter.prod.codeforafrica.org/charterafrica"
+  DOKKU_REMOTE_URL: "ssh://dokku@ui-1.prod.codeforafrica.org/charterafrica-staging"
```
Comment (kilemensi) [inline .github/workflows/charterafrica-deploy-prod.yml:13]: I take it the dokku app is indeed called `charterafrica-staging`?

**Review by kilemensi — APPROVED** [review]
🚀

File: .github/workflows/charterafrica-deploy-prod.yml (hunk 5 above)
Comment (thepsalmist) [reply .github/workflows/charterafrica-deploy-prod.yml:13]: Yes `charterafrica-staging`

**Pull request merged**